### PR TITLE
add navbar 2

### PR DESCRIPTION
**Pull request by MrAndriy**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ function App() {
   return (
     <div className="App">
       <h1>React Booking App</h1>
+      <h2>NavBar</h2>
       <p>some desc</p>
```
Comment (DadaLV) [inline client/src/App.js:6]: need more comments

**Review by DadaLV — COMMENTED** [review]
not good

**Comment by MrAndriy** [comment]
all good

**Review by DadaLV — APPROVED** [review]
nice

**Pull request merged**